python/feathub/feature_views/transforms/tests/test_expression_transform.py

```diff
@@ -340,3 +340,113 @@ def test_duplicate_feature_name_with_source(self):
         )
 
         self.assertTrue(expected_result_df.equals(result_df))
+
+    def test_map(self):
+        input_data = self.input_data.copy().head(2)
+        source = self.create_file_source(input_data)
+
+        feature_view_1 = DerivedFeatureView(
+            name="feature_view_1",
+            source=source,
+            features=[
+                Feature(
+                    name="row_map",
+                    transform="MAP('name', name, "
+                    "'cost', JSON_STRING(cost), "
+                    "'distance', JSON_STRING(distance), "
+                    "'time', time)",
+                ),
+                Feature(
+                    name="cost_distance_map",
+                    transform="MAP('cost', cost, 'distance', distance)",
+                ),
```

Comment:
Could you please also add a Feature whose `transform="MAP()"`? Zero arguments is also an even number of arguments.

Reply:
The problem with `MAP()` is that we cannot derive the type information of the result. Given that we don't see the use case at the moment, I think it is simpler that the `MAP` function requires at least two arguments.